dev/ledger.md

```diff
@@ -551,7 +551,9 @@ offline as part of applying the block changes to the ledger.
 A light block header is a structure contains a subset of fields from Algorand's  _block header_.
 Light block header contains the following components:
 
-- The block's _seed_, under msgpack key `s`.
+- The block's _seed_, under msgpack key `0`.
+
+- The block's _hash_, under msgpack key `1`. This represents a cryptographic hash of the block header.
```

Comment:
I'm assuming the other fields did not explain themselves because they are just copies of something in the BlockHeader. I think we could do the same here, not need to explain what it is.